src/builder.rs

```diff
@@ -137,6 +139,23 @@ impl SessionBuilder {
         self
     }
 
+    /// Enable or disable compression (including stdin, stdout, stderr, data
+    /// for forwarded TCP and unix-domain connections, sftp and scp
+    /// connections).
+    ///
+    /// NOTE that the ssh server can forcibly disable the compression.
+    ///
+    /// By default, ssh uses configure value set in `~/.ssh/config`.
+    ///
+    /// If `~/.ssh/config` does not enable compression, then it is disabled
+    /// by default.
+    ///
+    /// Defaults to `None`.
```

Comment:
This last point doesn't make sense I don't think? You already say what the default is above. There is no `Option` involved here.

Reply:
Fixed!